src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -521,6 +525,9 @@ void style_transfer::perform_predict(gl_sarray data, gl_sframe_writer& result,
 
   // looping through all of the style indices
   for (size_t i : style_idx) {
+    // check whether the style indicies are valid
```

Comment:
Check spelling: "indices"

Reply:
I don't know how I passed 2nd grade. LMAO. Thanks!

Reply:
Don't worry, no 2nd grader could do what you do!